src/core/agent/services/identifierService.ts

```diff
@@ -67,6 +67,10 @@ class IdentifierService extends AgentService {
     "Cannot join multi-sig inception as we do not control any member AID of the multi-sig";
   static readonly UNKNOWN_AIDS_IN_MULTISIG_ICP =
     "Multi-sig join request contains unknown AIDs (not connected)";
+  static readonly MISSING_GROUP_METADATA =
+    "Metadata record for group is missing";
+  static readonly ONLY_ALLOW_LINKED_CONTACTS =
+    "Only allow to create multisig with linked contacts";
```

Comment:
"Only allowed to create multi-sig using contacts with a matching groupId to our member identifier"